pkg/storage/driver/secrets.go

```diff
@@ -175,11 +184,16 @@ func (secrets *Secrets) Update(key string, rls *rspb.Release) error {
 	// create a new secret object to hold the release
 	obj, err := newSecretsObject(key, rls, lbs)
 	if err != nil {
+		secrets.Log("update: failed to encode release %q: %s", rls.Name, err)
 		return errors.Wrapf(err, "update: failed to encode release %q", rls.Name)
 	}
 	// push the secret object out into the kubiverse
 	_, err = secrets.impl.Update(context.Background(), obj, metav1.UpdateOptions{})
-	return errors.Wrap(err, "update: failed to update")
```

Comment:
This is the only major behavioural change I noticed in this PR. Have you tested to make sure this doesn't break backwards compatibility, for example with `helm upgrade`? A unit test to cover this would be great to see.

Reply:
Yep, I have test this and it works ok. AS for unit test, I think the existed tests have covered this.